sql/parse/parse.go

```diff
@@ -4143,6 +4143,48 @@ func ExprToExpression(ctx *sql.Context, e sqlparser.Expr) (sql.Expression, error
 			return nil, err
 		}
 		return function.NewExtract(unit, expr), err
+	case *sqlparser.MatchExpr:
```

Comment:
this file will be deprecated soon, would be helpful to also add this to `buildIndexDefs`